frontend/src/types/board/board.ts

```diff
@@ -39,6 +39,7 @@ export default interface BoardType {
 	postAnonymously: boolean;
 	submitedByUser?: string;
 	submitedAt?: Date;
+	slackGroup: boolean;
```

Comment:
Make `slackGroup` as a optional field.
```suggestion
	slackGroup?: boolean;
```